<ui version="4.0" >
 <class>EditItemDialogClass</class>
 <widget class="QDialog" name="EditItemDialogClass" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit item</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="lblLabel" >
     <property name="text" >
      <string>Edit item</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lneEdit" />
   </item>
   <item>
    <widget class="QGroupBox" name="grbTags" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btbButtons" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>btbButtons</sender>
   <signal>accepted()</signal>
   <receiver>EditItemDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>26</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel" >
     <x>4</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btbButtons</sender>
   <signal>rejected()</signal>
   <receiver>EditItemDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>155</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel" >
     <x>174</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
